pdx-extract/app/main.py

```diff
@@ -0,0 +1,174 @@
+import time
+
+import requests
+
+from app import config
+from app import pyppeteer
+from app import sanitize
+from app import storage
+from app import urls
```

Comment:
Okay, first code review here from someone who's never graced industry, so be nice (I know you will) :D.

Personal opinion you will most likely disagree with: keeping multiple imports from one library on one line (within reason) is just as (if not more) readable than having to re-read `from app import ...` multiple times.
So it would be:
```
from app import config, pyppeteer...
```

Reply:
Good question that led me down a rabbit hole!

[PEP8](https://www.python.org/dev/peps/pep-0008/#imports) seems to imply that imports should be on separate lines. However, I ran both suggestions through my two formatters (flake8 and black) and neither of them complained, so I'm not 100% sure which is the more pythonic approach.

Reply:
I don't know about Pythonic, but as far as using tools such as `git blame` it's definitely better to split them into separate lines, so each package has its own committer.

Reply:
That was my original rationale as well.

Reply:
Oh very interesting! I really should read these guides more. A great resource. Thank you.